### PR TITLE
Fix Inaccurate Estimated Interest Rate

### PR DESCRIPTION
**Pull request by Drypto13**

### CODE REVIEW

**Comment by RomanHiden** [comment]
we need to add testcase other than that looks good

File: contracts/connectors/loantoken/LoanTokenLogicStandard.sol [hunk 1]
```diff
@@ -1004,6 +1004,24 @@ contract LoanTokenLogicStandard is AdvancedToken, StorageExtension {
 
     /* Internal View functions */
 
+    function _nextBorrowInterestRate(
```
Comment (RomanHiden) [inline contracts/connectors/loantoken/LoanTokenLogicStandard.sol:1006]: there is original `function _nextBorrowInterestRate(` if its not needed we remove. if its needed we call new one from there

Comment (Drypto13) [reply contracts/connectors/loantoken/LoanTokenLogicStandard.sol:1006]: the original one is needed as called by protocol contract. This one is internal and only used for proper estimations.

Comment (RomanHiden) [reply contracts/connectors/loantoken/LoanTokenLogicStandard.sol:1006]: looks good now

**Review by RomanHiden — APPROVED** [review]
I am approving this. but it has to go with the itoken-collateral probably

**Pull request merged**